iocage/cli/create.py

```diff
@@ -53,21 +53,28 @@ def validate_count(ctx, param, value):
 
 @click.command(name="create", help="Create a jail.")
 @click.option("--count", "-c", callback=validate_count, default="1")
-@click.option("--release", "-r", required=False)
-@click.option("--template", "-t", required=False)
+@click.option("--release", "-r", required=False,
+              help="Specify the release to use for the new jail.")
+@click.option("--template", "-t", required=False,
+              help="Flag this jail as a template, which allows for rapid "
+                   "redeployment of a customized jail.")
 @click.option("--pkglist", "-p", default=None)
 @click.option("--name", "-n", default=None,
               help="Provide a specific name and tag instead of an UUID for"
-                   " this jail")
+                   " this jail.")
 @click.option("--uuid", "-u", "_uuid", default=None,
-              help="Provide a specific UUID for this jail")
-@click.option("--basejail", "-b", is_flag=True, default=False)
-@click.option("--empty", "-e", is_flag=True, default=False)
+              help="Provide a specific UUID for this jail.")
+@click.option("--basejail", "-b", is_flag=True, default=False,
+              help="Set the new jail type to a basejail. Basejails "
+                   "re-clone their base filesystems at each startup.")
```

Comment:
This is only true for the legacy implementation. The new one actually mounts the specified `RELEASEs` directories as `nullfs` mounts over the jails directories.

So `11.0-RELEASE/root/bin` will be mounted as a R/O nullfs mount over `foo/root/bin`, saving space and making moving to a new `RELEASE` much faster. Once the upgrade mechanism is figured out and implemented that is ;)